src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -745,13 +803,15 @@ private String getUpToDateAccessURL(String resourceIdentifier, String accessURL)
 		// If we got a URL back from identifiers.org...
 		if (urlFromIdentifiersDotOrg != null && !urlFromIdentifiersDotOrg.trim().equals(""))
 		{
-			if (!urlFromIdentifiersDotOrg.replace("{$id}", "").equals(accessURL.replace("###ID###", "")))
+			final String identifiersDotOrgIDToken = "{$id}";
+			final String reactomeIDToken = "###ID###";
+			if (!urlFromIdentifiersDotOrg.replace(identifiersDotOrgIDToken, "").equals(accessURL.replace(reactomeIDToken, "")))
 			{
 				// If replacing the Identifier tokens cause the two strings to mis-match, we should
 				// use the new accessURL from identifiers.org, and log a message so someone will
 				// know to update reference-databases.xml
-				updatedAccessURL = urlFromIdentifiersDotOrg.replace("{$id}", "###ID###");
-				logger.info("accessURL changed from: '{}' to: '{}', as per the data at identifiers.org", accessURL, urlFromIdentifiersDotOrg.replace("{$id}", "###ID###"));
+				updatedAccessURL = urlFromIdentifiersDotOrg.replace(identifiersDotOrgIDToken, reactomeIDToken);
+				logger.info("accessURL changed from: '{}' to: '{}', as per the data at identifiers.org", accessURL, updatedAccessURL);
```

Comment:
I never realized I should update the reference-databases.xml file based on this message -- perhaps that should in the the info message?